### PR TITLE
Review environment added to template

### PR DESCRIPTION
**Pull request by VenkataChalla554**
## Context
Added review environment  changes to new_service  template

## Changes proposed in this pull request
Changes done to make file , added  review.tfvars , review_Terrafile and review.sh under templates/new_service.

## Guidance to review
Added files should be available when you generate new service .
 
## Checklist

- [x] I have performed a self-review of my code, including formatting and typos
- [x] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [x] I have added the `Devops` label
- [x] I have attached the pull request to the trello card

### CODE REVIEW

**Review by johnake — APPROVED** [review]

File: templates/new_service/terraform/application/config/review.tfvars.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "cluster": "test",
+    "namespace": "#NAMESPACE_PREFIX#-development",
+    "environment": "development",
```
Comment (saliceti) [inline templates/new_service/terraform/application/config/review.tfvars.json:4]: environment is dynamic for review apps so it should not be here

File: templates/new_service/terraform/application/config/review_Terrafile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+aks:
+    source:  "https://github.com/DFE-Digital/terraform-modules"
+    version: "testing"
```
Comment (saliceti) [inline templates/new_service/terraform/application/config/review_Terrafile:3]: Review apps can pick up terraform-modules changes without risk, please set it to "main"

File: templates/new_service/Makefile [hunk 3]
```diff
@@ -13,6 +13,13 @@ help:
 development: test-cluster
 	$(eval include global_config/development.sh)
 
+.PHONY: review
+review: test-cluster
+	$(if ${PR_NUMBER},,$(error Missing PR_NUMBER))
+	$(eval ENVIRONMENT=${PR_NUMBER})
+	$(eval TF_VAR_ENVIRONMENT=${PR_NUMBER})
```
[the remark this answers is not in the document]
Comment (VenkataChalla554) [reply templates/new_service/Makefile:20]: Yes, TF_VAR_environment not required.

File: templates/new_service/terraform/application/config/review.tfvars.json (hunk 1 above)
Comment (VenkataChalla554) [reply templates/new_service/terraform/application/config/review.tfvars.json]: Removed environment

File: templates/new_service/Makefile [hunk 4]
```diff
@@ -13,6 +13,12 @@ help:
 development: test-cluster
 	$(eval include global_config/development.sh)
 
+.PHONY: review
+review: test-cluster
+	$(if ${PR_NUMBER},,$(error Missing PR_NUMBER))
+	$(eval ENVIRONMENT=${PR_NUMBER})
```
Comment (saliceti) [inline templates/new_service/Makefile:19]: Don't we need TF_VAR_environment to pass the value to terraform?

**Pull request merged**